src/cosmology/compat/camb/__init__.py

```diff
@@ -17,6 +17,10 @@
     Array: TypeAlias = NDArray[np.float64]
 
 
+K_LINEAR: float = 1e-3
```

Comment:
```suggestion
K_LINEAR: Final = 1e-3
```

Reply:
I actually made it a module constant so that it can be changed by the user!